crud.py

```diff
@@ -0,0 +1,53 @@
+from sqlalchemy import select
+from sqlalchemy.orm import Session
+
+import schemas
+from db import models
+
+
+def get_all_authors_paginated(db: Session, page: int = 0, size: int = 10):
+    offset = page * size
+    selected_authors = select(models.Author).offset(offset).limit(size)
```

Comment:
The `select` method is used here, but it should be combined with `db.execute()` as it is in line 11. Ensure that the `select` statement is correctly constructed to match the ORM model usage.